Scala/syntax_test_scala.scala

```diff
@@ -306,6 +306,10 @@ type Foo = Bar[A] forSome { type A }
 // <- comment.block.documentation.scala
 */
 
+  /**/0xff
+//^^^^ comment.block.empty.scala
+//    ^^^^ - thereafter, not a comment 
```

Comment:
Thanks for explaining and pointing me in the direction of the documentation I should have read :)

Will amend the PR tomorrow.